backend/src/types.ts

```diff
@@ -11,12 +11,19 @@ type BoardState = {
   players: string[];
 };
 
+interface LoginHandlerParams {
```

Comment:
I'm not sure if this is the best way to layout the types. I think it might be better to create nested types rather than duplicating so many common parameters. I am going to ask Justin for more info about this!

Reply:
we might be able to do something like
```
interface LoginHandlerParams {
  ...
}
interface HandlerParams extends LoginHandlerParams {
  username: string;
}
```

Reply:
Yeah this is what I was thinking too. Let's do that but just provide some better naming. Call LoginHandlerParams something more generic like BaseHandlerParams. Then HandlerParams could be named something like GameHandlerParams because it also has a username. Then for all our future unique handlers we can extend BaseHandlerParams.